Rakefile

```diff
@@ -6,19 +6,10 @@ require "active_record"
 namespace :db do
   db_config = YAML.safe_load(File.open("spec/config/database.yml"))
 
-  def activerecord_below_5_2?
-    ActiveRecord.version.release < Gem::Version.new("5.2.0")
-  end
-
   desc "Migrate the database"
   task :migrate do
     ActiveRecord::Base.establish_connection(db_config)
-
-    if activerecord_below_5_2?
-      ActiveRecord::Migrator.migrate("spec/db/migrate")
-    else
-      ActiveRecord::MigrationContext.new("spec/db/migrate").migrate
-    end
```

Comment:
We are removing this so as to not support <5.2?

Reply:
We're only ever going to run this on Panacea, which is Rails 7, so I figured it wouldn't hurt to take that out.

Reply:
I agree 👍